src/tests/unit-tests/helpers.test.ts

```diff
@@ -0,0 +1,47 @@
+import { validateInput, shortenPublicHoliday } from '../../helpers';
+
+const testData = {
+  year: 2024,
+  country: 'FR'
+};
+
+const testPublicHoliday = {
+  date: '21.03.2024',
+  localName: 'Thanksgiving day',
+  name: 'test',
+  countryCode: 'test',
+  fixed: true,
+  global: false,
+  countries: ['FR'],
+  launchYear: 1955,
+  types: ['test']
+};
+
+describe('test validateInput function', () => {
+  test('shoud return true if data is correct', () => {
+    expect(validateInput(testData)).toBeTruthy();
+  });
+
+  test('shoud return error if year is not current', () => {
+    testData.year = 2020;
```

Comment:
```suggestion
    const localTestData = {
      ...testData,
      year: 2020
    };
```
It's better not to modify test data which is used in many tests not to get unexpectedly false results.